Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-visualsearch/README.md

#### nodejs - Authentication, client creation and visualSearch images as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

```typescript
import {
  VisualSearchClient,
  VisualSearchModels
} from "@azure/cognitiveservices-visualsearch";
import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";

async function main(): Promise<void> {
  const visualSearchKey = process.env["visualSearchKey"] || "<visualSearchKey>";
  const visualSearchEndPoint =
    process.env["visualSearchEndPoint"] || "<visualSearchEndPoint>";
  const cognitiveServiceCredentials = new CognitiveServicesCredentials(
    visualSearchKey
  );
  const client = new VisualSearchClient(cognitiveServiceCredentials, {
    endpoint: visualSearchEndPoint
  });

  const insightsToken =

>     "ccid_tmaGQ2eU*mid_D12339146CFEDF3D409CC7A66D2C98D0D71904D4*simid_608022145667564759*thid_OIP.tmaGQ2eUI1yq3yll!_jn9kwHaFZ";

Reviewer: This shouldnt be hard coded

Author: It is a generic publicly available token taken from https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/cognitive-services/bing-visual-search/use-insights-token.md. 

I have removed it now

#### nodejs - Authentication, client creation and visualSearch images as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

> The following sample lets you perform a visual search, i.e. perform a search with a image. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/bing-visual-search/).

Reviewer: ```suggestion
The following sample performs a visual search, i.e. perform a search with a image. To know more, refer to the [Azure Documentation on Bing Visual Search](https://docs.microsoft.com/en-us/azure/cognitive-services/bing-visual-search/).
```
